OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/OoyalaSkinPlayerActivity.java

```diff
@@ -35,7 +35,7 @@ void completePlayerSetup(boolean asked) {
 			skinLayout = (OoyalaSkinLayout) findViewById(R.id.ooyalaSkin);
 			// Create the OoyalaPlayer, with some built-in UI disabled
 			PlayerDomain playerDomain = new PlayerDomain(domain);
-			Options options = new Options.Builder().setShowNativeLearnMoreButton(false).setShowPromoImage(false).setUseExoPlayer(true).build();
+			Options options = new Options.Builder().setShowNativeLearnMoreButton(false).setShowPromoImage(false).setUseExoPlayer(true).setBypassPCodeMatching(true).build();
```

Comment:
No need to add bypass by default, just make sure you add the correct pcodes with embed codes. If you don't know how to get the pcode let me know.

Reply:
I took, for instance, DRM DASH from this list https://docs.google.com/spreadsheets/d/1EWh5a5UOgLHlZ78OdBGOKuVjwoed14Nms0q_BsrIOvs/edit#gid=1633781289 (the first link TC01).
Embed code Q1cG85NTE6Df3A95XMMbKGsPg6yaEZGm
Pcode FoeG863GnBL4IhhlFC1Q2jqbkH9m
And than I got the error: "The provided pcode does not match the embed code owner". Is the pcode correct?
I guess I chose the correct one.

Reply:
Also eHLS doesn't work without setBypassPCodeMatching(true).
Res http://debug.ooyala.com/ea/index.html?ec=hneHVnMzE6sEsCT9DE1v_yWi0o-9aCTw&pbid=378512f55dd44fc287d0fa609353c268&pcode=x2aDkyOt2q3WtOCp-krSyDffASzL&core_player=http%3A%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fcandidate%2Flatest%2Fcore.js&video_plugins=http%3A%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fcandidate%2Flatest%2Fvideo-plugin%2Fbit_wrapper.min.js&html5_skin=http%3A%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fcandidate%2Flatest%2Fskin-plugin%2Fhtml5-skin.min.js&skin_asset=http%3A%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fcandidate%2Flatest%2Fskin-plugin%2Fhtml5-skin.min.css&skin_config=http%3A%2F%2Fplayer.ooyala.com%2Fstatic%2Fv4%2Fcandidate%2Flatest%2Fskin-plugin%2Fskin.json&options=%7B%22platform%22%3A%22html5%22%7D